### ISSUE TITLE
rvm.ruby task fails for 3 VMs

### ISSUE THREAD

**Issue by meetinthemiddle-be** [hysnsec/DevSecOps-Studio#87]
Fails for DevSecOps-Box, jenkins, gitlab-runner VMs

Example output for rvm.ruby task on DevSecOps-Box; similar for other two

```TASK [rvm.ruby : Detect rvm binary] ********************************************
ok: [DevSecOps-Box]

TASK [rvm.ruby : Detect rvm installer] *****************************************
ok: [DevSecOps-Box]

TASK [rvm.ruby : Detect current rvm version] ***********************************
skipping: [DevSecOps-Box]

TASK [rvm.ruby : Install rvm installer] ****************************************
changed: [DevSecOps-Box]

TASK [rvm.ruby : Import GPG keys] **********************************************
ok: [DevSecOps-Box]

TASK [rvm.ruby : Import GPG keys the other way] ********************************
skipping: [DevSecOps-Box] => (item=mpapis.asc) 
skipping: [DevSecOps-Box] => (item=pkuczynski.asc) 

TASK [rvm.ruby : Install rvm] **************************************************
changed: [DevSecOps-Box]

TASK [rvm.ruby : Update rvm] ***************************************************
skipping: [DevSecOps-Box]

TASK [rvm.ruby : Configure rvm] ************************************************
changed: [DevSecOps-Box]

TASK [rvm.ruby : Detect if rubies are installed] *******************************
[DEPRECATION WARNING]: evaluating ['ruby-2.3.1'] as a bare variable, this 
behaviour will go away and you might need to add |bool to the expression in the
 future. Also see CONDITIONAL_BARE_VARS configuration toggle.. This feature 
will be removed in version 2.12. Deprecation warnings can be disabled by 
setting deprecation_warnings=False in ansible.cfg.
ok: [DevSecOps-Box] => (item=ruby-2.3.1)

TASK [rvm.ruby : Install rubies] ***********************************************
changed: [DevSecOps-Box] => (item={'changed': False, 'end': '2020-01-04 15:20:53.265741', 'stdout': '', 'cmd': ['/usr/local/rvm/bin/rvm', 'ruby-2.3.1', 'do', 'true'], 'failed': False, 'delta': '0:00:00.271021', 'stderr': 'Ruby ruby-2.3.1 is not installed.', 'rc': 2, 'invocation': {'module_args': {'creates': None, 'executable': None, '_uses_shell': False, 'strip_empty_ends': True, '_raw_params': '/usr/local/rvm/bin/rvm ruby-2.3.1 do true', 'removes': None, 'argv': None, 'warn': True, 'chdir': None, 'stdin_add_newline': True, 'stdin': None}}, 'start': '2020-01-04 15:20:52.994720', 'msg': 'non-zero return code', 'stdout_lines': [], 'stderr_lines': ['Ruby ruby-2.3.1 is not installed.'], 'failed_when_result': False, 'item': 'ruby-2.3.1', 'ansible_loop_var': 'item'})

TASK [rvm.ruby : Detect default ruby version] **********************************
ok: [DevSecOps-Box]

TASK [rvm.ruby : Select default ruby] ******************************************
changed: [DevSecOps-Box]

TASK [rvm.ruby : Detect installed ruby patch number] ***************************
ok: [DevSecOps-Box] => (item=ruby-2.3.1)

TASK [rvm.ruby : Install bundler if not installed] *****************************
failed: [DevSecOps-Box] (item={'changed': False, 'end': '2020-01-04 15:21:19.295819', 'stdout': 'ruby-2.3.1', 'cmd': '/usr/local/rvm/bin/rvm list strings | grep ruby-2.3.1 | tail -n 1\n', 'rc': 0, 'start': '2020-01-04 15:21:19.102294', 'stderr': '', 'delta': '0:00:00.193525', 'invocation': {'module_args': {'creates': None, 'executable': None, '_uses_shell': True, 'strip_empty_ends': True, '_raw_params': '/usr/local/rvm/bin/rvm list strings | grep ruby-2.3.1 | tail -n 1\n', 'removes': None, 'argv': None, 'warn': True, 'chdir': None, 'stdin_add_newline': True, 'stdin': None}}, 'stdout_lines': ['ruby-2.3.1'], 'stderr_lines': [], 'failed': False, 'item': 'ruby-2.3.1', 'ansible_loop_var': 'item'}) => {"ansible_loop_var": "item", "changed": false, "cmd": "ls /usr/local/rvm/wrappers/ruby-2.3.1 | if ! grep \"^bundler \" ; then /usr/local/rvm/wrappers/ruby-2.3.1/gem install bundler ; fi\n", "delta": "0:00:00.685977", "end": "2020-01-04 15:21:20.441122", "item": {"ansible_loop_var": "item", "changed": false, "cmd": "/usr/local/rvm/bin/rvm list strings | grep ruby-2.3.1 | tail -n 1\n", "delta": "0:00:00.193525", "end": "2020-01-04 15:21:19.295819", "failed": false, "invocation": {"module_args": {"_raw_params": "/usr/local/rvm/bin/rvm list strings | grep ruby-2.3.1 | tail -n 1\n", "_uses_shell": true, "argv": null, "chdir": null, "creates": null, "executable": null, "removes": null, "stdin": null, "stdin_add_newline": true, "strip_empty_ends": true, "warn": true}}, "item": "ruby-2.3.1", "rc": 0, "start": "2020-01-04 15:21:19.102294", "stderr": "", "stderr_lines": [], "stdout": "ruby-2.3.1", "stdout_lines": ["ruby-2.3.1"]}, "msg": "non-zero return code", "rc": 1, "start": "2020-01-04 15:21:19.755145", "stderr": "ERROR:  Error installing bundler:\n\tbundler requires RubyGems version >= 2.5.2. Try 'gem update --system' to update RubyGems itself.", "stderr_lines": ["ERROR:  Error installing bundler:", "\tbundler requires RubyGems version >= 2.5.2. Try 'gem update --system' to update RubyGems itself."], "stdout": "", "stdout_lines": []}
```

**Comment by robbyprescott** [comment]
Same issue in February 2020.

**Comment by secfigo** [comment]
Fixed in the #90 

Please pull and use it.
